be/src/exprs/vectorized/runtime_filter_bank.h

```diff
@@ -24,11 +24,12 @@ class RowDescriptor;
 class MemTracker;
 class ExecEnv;
 class RuntimeProfile;
-
+namespace pipeline {
+class RuntimeBloomFilterEvalContext;
```

Comment:
`RuntimeBloomFilterEvalContext` is defined previously as a struct here but declared here as a class. Maybe we should unify them.

Reply:
ok